etna/search/templatetags/search_tags.py

```diff
@@ -198,3 +200,17 @@ def hidden_fields_for_date_filter(selected_filters, form) -> str:
             html += include_hidden_fields(visible_field_names_str, form)
 
     return mark_safe(html)
+
+
+@register.filter
+def search_title(search_tab) -> str:
+    """
+    Returns title for search tab
+    """
+    if search_tab == SearchTabs.ALL.value:
+        label = "All search results"
+    elif search_tab == SearchTabs.CATALOGUE.value:
+        label = "Catalogue search results"
+    elif search_tab == SearchTabs.WEBSITE.value:
+        label = "Website search results"
```

Comment:
Sorry I am slightly late on this, if none of the conditions are evaluated then the return is Non, don't we need an else here? also why we are not returned immediately rather than wait till the end, for example, if search_tab == SearchTabs.ALL.value:  return label, also no checking on the search_tab before usage

Reply:
**Yup, its late - Its merged already.**

> if none of the conditions are evaluated then the return is Non, don't we need an else here?

None/else is not evaluated in this case as input param come from  enum SearchTabs itself.

>  also why we are not returned immediately rather than wait till the end, 

1. Once the else condition is evaluated. the next step is return - its a small piece of code.
2. I thought it would be cleaner to have one return than multiple returns in the if-else conditions

Reply:
The only thing here is the full elifs will be evaluated before any return, but if one of them is true then just return immediately faster and less buggy :)

Reply:
> None/else is not evaluated in this case as input param come from enum SearchTabs itself.

Reply:
Not true, I can call the function passing None like :search_title(None) :), anyway the point I would like to make is we may need to be slightly more protective over our code.

Reply:
Noted. Thank you.